Scripts/build.sh

```diff
@@ -86,10 +86,15 @@ function xcbuild() {
 }
 
 function fastlane_test() {
+  if [[ ! $(which fastlane) ]] ; then
```

Comment:
I assume it should be  better to use [command -v](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script) for this purpose

Reply:
oh, cool. I didn't know this